azurerm/resource_arm_iothub.go

```diff
@@ -248,6 +248,12 @@ func resourceArmIotHub() *schema.Resource {
 				},
 			},
 
+			"enable_fallback_route": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
For now we should keep the default terraform uses as to not break anything, and then when 2.0 comes out we can change the default.
```suggestion
				Default:  false, // todo change to true in 2.0
```

Reply:
I understand, modified the default to false.